packages/main/src/i18n/messagebundle.properties

```diff
@@ -416,7 +416,7 @@ TOKENIZER_ARIA_CONTAIN_SEVERAL_TOKENS=Contains {0} tokens
 TOKENIZER_ARIA_LABEL=Tokenizer
 
 #XFLD: Remove label for Tokenizer's Dialog on phone
```

Comment:
You should change the naming and description as the translation team uses the description to determine the right translation for a property.